packages/datadog-instrumentations/src/aerospike.js

```diff
@@ -40,7 +40,7 @@ function wrapProcess (process) {
 addHook({
   name: 'aerospike',
   file: 'lib/commands/command.js',
-  versions: ['4', '5']
+  versions: ['^3.16.2', '4', '5']
```

Comment:
Didn't notice this before, but this caps the version to v5, so when v6 is released the instrumentation will stop working.